README.md

```diff
@@ -1,71 +1,86 @@
+- Message signing and verification with automatically generated RSA keys.
+- Message integrity validation (via including the digests of the previous messages).
+- Authentication of each command/message by SMP servers with automatically generated RSA key pairs.
 - TCP transport encryption using SMP transport protocol.
 
 RSA keys are not used as identity, they are randomly generated for each contact. 2048 bit keys are used, it can be changed to 4096-bit in code via [rsaKeySize setting](https://github.com/simplex-chat/simplex-chat/blob/master/apps/dog-food/Main.hs).
 
-The chat client is stable, but it is quite new and unaudited, so you probably should NOT use it yet for high security communications - unless you know what you are doing.
+## Installation
 
-## SimpleX chat roadmap for 2021-22
+### Download executable chat client
 
-1. Switch to application level chat protocol. This will allow to separate physical server connection management from logical chat contacts, and to support all common chat functions.
-2. SMP queue redundancy and rotation in SMP agent protocol.
-3. Symmetric groups support in SMP agent protocol, as a foundation for chat groups.
-4. Delivery confirmation in SMP agent protocol.
-5. Multi-agent/device data synchronisation - to use chat on multiple devices.
-6. Synchronous streams support in SMP and SMP agent protocols, to support file transfer.
-7. Terminal chat UI and mobile apps.
-8. Scripts for simple SMP server deployment to hosting providers: Linode, Digital Ocean and Heroku.
-9. Public broadcast channels.
-10. Optional public contact/group addresses using DNS to establish connections, but not using it to send and receive messages - in this way you will keep all your contacts and groups even if you lose the control of the domain.
+Download the chat binary for your system from the [latest stable release](https://github.com/simplex-chat/simplex-chat/releases) and make it executable as shown below.
 
-## Installing the chat client
+#### Linux and MacOS
 
-You can:
-- download the executable file built on Github CI
-- build it from source
+```sh
+chmod +x <binary>
+mv <binary> ~/.local/bin/dog-food
+```
+
+(or any other preferred location on PATH).
 
-### Downloading the executable chat client
+On MacOS you also need to [allow Gatekeeper to run it](https://support.apple.com/en-us/HT202491).
 
-You can download the executable binary file for your platform from the [latest stable release](https://github.com/simplex-chat/simplex-chat/releases) and make it executable:
+#### Windows
 
-  - Linux and MacOS: `chmod +x <binary>; mv <binary> ~/.local/bin/dog-food` (or any other preferred location on PATH). On MacOS you need to [allow Gatekeeper to run it](https://support.apple.com/en-us/HT202491).
-  - Windows: `move <binary> %APPDATA%\local\bin\dog-food.exe`.
+```sh
+move <binary> %APPDATA%\local\bin\dog-food.exe
```

Comment:
let's change this to forward slashes for consistency with `%APPDATA%/simplex` below - Windows now accepts forward slashes in many cases